README.md

```diff
@@ -16,10 +16,11 @@
 | \- Financial transactions: Requires ACID compliance.<br>\- Complex queries: Needs structured and relational data.                  | \- Design a financial transaction system.<br>\- Create a scalable relational database.                     | **SQL Database**                    | Strong ACID properties, structured data, complex queries.   | Limited scalability, schema management.             | \- Implement sharding.<br>\- Use read replicas.<br>\- Employ clustering and partitioning.                                                          | MySQL, PostgreSQL, MS SQL Server            |
 | \- Large-scale data: Supports horizontal scaling.<br>\- Unstructured data: Flexible schema adapts to changes.                      | \- Design a large-scale user profile store.<br>\- Create a scalable data storage solution.                 | **NoSQL Database**                  | Flexible schema, horizontal scalability, high performance.  | Eventual consistency, limited transaction support.  | \- Use consistency settings (e.g., quorum reads/writes).<br>\- Design for idempotent operations.<br>\- Implement conflict resolution strategies.   | MongoDB, Cassandra, DynamoDB                |
-| \- High availability: Ensures data is replicated and available.<br>\- Disaster recovery: Provides data redundancy.                 | \- Design a data replication strategy.<br>\- Implement a highly available database system.                 | **Replication Service**             | Ensures data availability and redundancy.                   | Increases costs, consistency issues.                | \- Use asynchronous replication.<br>\- Implement conflict resolution.<br>\- Use multi-master replication.                                          | AWS RDS Read Replicas, MongoDB Replica Sets |
+| \- High availability: Ensures data is replicated and available. | \- Design a data replication strategy.<br>\- Implement a highly available database system.                 | **Data Replication**             | Ensures data durability, to ensure system availability.                   | Increases costs, consistency issues.                | \- Use asynchronous replication.<br>\- Implement conflict resolution.<br>\- Use multi-master replication.                                          | AWS RDS standby (synchronous), AWS RDS Read Replicas (asynchronous), MongoDB Replica Set (asynchronous)|
+| \- Disaster recovery: Provides data protection.                 | \- Design a data backup strategy.<br>\- Implement a disaster recovery plan.                 | **Data Backup**             | Ensures data recoverability.                   | Increases costs, if backups are not up to date there will be data loss, backup may not be accessible.                | \- Use continuous or automated backup.<br>\- Backup to another region. resolution.                                          | Use native backup capabilites of the data store, or centralized backup products like AWS Backup, Veeam, or ARPIO|
```

Comment:
Hey this one is already present as "Backup and Recovery" component. So could you remove this "Disaster recovery" row? 
Rest additions/edits look good to me. :)

Reply:
ok, done!